sdk/va-report-components/documentation/docs/api/ObjectHandle.md

```diff
@@ -38,3 +38,43 @@ If no `options` parameter is supplied, the report is exported using the default
 
 Refreshes the data for the report object that is controlled by the
 `ObjectHandle`.
+
+### getSelectedData(options?: {formatData: boolean | "datesOnly"}): ReportObjectResultData[]
+
+Returns a user's selection data from the visual. Returns an empty array if the visual has no selections.
```

Comment:
To be consistent with the VA doc, we're using the term "object" instead of "visual".